figures/scripts/transcriptomic-overview.R

```diff
@@ -0,0 +1,336 @@
+  plot_dimension_reduction(point_color = "short_histology",
+                           x_label = "UMAP1",
+                           y_label = "UMAP2",
+                           color_palette = histology_palette) +
+  theme(text = element_text(size = 15))
+
+# Save temporary PNG
+ggsave(umap_png, plot = umap_plot, width = 4, height = 4)
+
+#### GSVA scores ---------------------------------------------------------------
+
+gsva_dir <- file.path(
+  analyses_dir,
+  "gene-set-enrichment-analysis"
+)
+
+gsva_scores_file <- file.path(
+  gsva_dir,
+  "results",
+  "gsva_scores_stranded.tsv"
+)
+
+gsva_stats_file <- file.path(
+  gsva_dir,
+  "results",
+  "gsva_anova_stranded_short_histology.tsv"
+)
+
+gsva_scores_df <- read_tsv(gsva_scores_file)
+gsva_stats_df <- read_tsv(gsva_stats_file)
+
+gsva_scores_mat <- gsva_scores_df %>%
+  spread(Kids_First_Biospecimen_ID, gsea_score) %>%
+  mutate(hallmark_name = stringr::str_replace_all(
+    stringr::str_remove(hallmark_name, "HALLMARK_"), "_", " ")
+  ) %>%
+  tibble::column_to_rownames("hallmark_name") %>%
+  as.matrix()
+
+# What gene sets are we going to include in our heatmap?
+# We'll filter based on whether the ANOVA has a significant p-value
+included_genesets <- gsva_stats_df %>%
+  mutate(hallmark_name = stringr::str_replace_all(
+    stringr::str_remove(hallmark_name, "HALLMARK_"), "_", " ")
+  ) %>%
+  filter(significant_anova) %>%
+  pull(hallmark_name)
+
+# Filter to only those gene sets
+gsva_scores_mat <- gsva_scores_mat[included_genesets, ]
+
+# Let's deal with the color palette for the heatmap we will make
+na_color <- divergent_palette %>%
+  filter(color_names == "na_color")
+
+divergent_palette  <- divergent_palette %>%
+  filter(color_names != "na_color")
+
+divergent_col_val <- seq(from = min(gsva_scores_mat),
+                         to = max(gsva_scores_mat),
+                         length.out = nrow(divergent_palette))
+
+gsva_col_fun <- circlize::colorRamp2(divergent_col_val,
+                                     divergent_palette$hex_codes)
+
+# Annotation bar for short histology
+# First get a data frame for the annotation and ensure samples are in the
+# same order
+short_histology_df <- histologies_df %>%
+  filter(experimental_strategy == "RNA-Seq",
+                RNA_library == "stranded") %>%
+  select(Kids_First_Biospecimen_ID,
+                short_histology) %>%
+  arrange(Kids_First_Biospecimen_ID) %>%
+  tibble::column_to_rownames("Kids_First_Biospecimen_ID") %>%
+  as.data.frame()
+
+gsva_scores_mat <- gsva_scores_mat[, order(colnames(gsva_scores_mat))]
+
+# The colors supplied to HeatmapAnnotation need to be a named
+# list of named vectors
+annotation_colors <- list(
+  short_histology = tibble::deframe(histology_palette)
+)
+
+# Annotation bar intended for the top of the heatmap
+column_heatmap_annotation <- HeatmapAnnotation(
+  df = short_histology_df,
+  col = annotation_colors,
+  name = "short histology",
```

Comment:
Is it possible to move the name to the left? It kind of looks like it is the name for the row labels in its current position. Also, I note the included png does not reflect this name, but uses `short_histology` for some reason.